manifests/params.pp

```diff
@@ -21,20 +21,25 @@
   $use_scl_ruby = true
   $scl_ruby = 'rh-ruby25'
```

Comment:
Shouldn't this also be only on EL7?

Reply:
Theoretically, we allow users to pass this in for some reason so I left it alone.